packages/components/pie-switch/src/switch.scss

```diff
@@ -85,8 +85,17 @@
 }
 
 .c-switch-input {
-    appearance: none;
-    margin: 0;
+    position: absolute;
```

Comment:
Did we check this still looks okay in RTL?

Reply:
Nope - will do now

Reply:
Seems fine locally testing but I believe Percy will also catch any regression here